jekyll/_cci2/reusing-config.md

```diff
@@ -249,10 +249,9 @@ jobs:
     executor: baz-orb/bar  # prefixed executor
 ```
 
-Note that `foo-orb/bar` and `baz-orb/bar` are different executors. They
-both have the local name `bar` relative to their orbs, but the are independent executors living in different orbs.
+**Note:** The `foo-orb/bar` and `baz-orb/bar` are different executors. They
```

Comment:
Clarifications like this are where I think the test files from `orb-service` provide really rich examples. e.g. this [build.yml](https://github.com/circleci/orb-service/blob/master/server/resources/test/executor/single/build.yml) compiles to this [config.yml](https://github.com/circleci/orb-service/blob/master/server/resources/test/executor/single/config.yml)

Reply:
I'm not sure what addition you want me to make here?